digitalstrategie/templatetags/extra_tags.py

```diff
@@ -0,0 +1,11 @@
+import requests
+from django import template
+from django.utils.html import mark_safe
+
+register = template.Library()
+
+
+@register.simple_tag
+def get_external_footer():
+    response = requests.get('https://www.berlin.de/rbmskzl/__i9/ohne/foot.inc')
```

Comment:
I guess, this should go into a setting? @goapunk ?

Reply:
since we usually use a setting for external links (?) we could do that here too, yes